settings.py

```diff
@@ -39,6 +39,7 @@
 SECURITY_SEND_CONFIRMATION_TEMPLATE = 'auth/confirmation.html'
 EXPLAIN_TEMPLATE_LOADING = False
 
+DEBUG_TB_ENABLED = False
```

Comment:
I see why you did that and I tried (very!) hard to do that only for tests with no success 😢

Reply:
I'll suggest another PR with that